packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -190,6 +195,44 @@ export const visualizeSharedString: VisualizeSharedObject = async (
 	};
 };
 
+/**
+ * {@link VisualizeSharedObject} for {@link ISharedTree}.
+ */
+export const visualizeSharedTree: VisualizeSharedObject = async (
+	sharedObject: ISharedObject,
+	visualizeChildData: VisualizeChildData,
+): Promise<FluidObjectTreeNode> => {
+	const sharedTree = sharedObject as ISharedTree;
+
+	const contextRoot = sharedTree.context.root;
+	const children: Record<string, VisualChildNode> = {};
+
+	const iterateNodes = async (field: UntypedField): Promise<void> => {
+		for (const child of field) {
+			await iterateFields(child as unknown as UntypedTree);
+		}
+	};
+
+	const iterateFields = async (node: UntypedTree): Promise<void> => {
+		for (const field of node) {
+			const renderedChild = await visualizeChildData(field);
+
+			children[field.fieldKey as string] = renderedChild;
+		}
+	};
+
+	if (sharedTree.root !== undefined) {
```

Comment:
Just curious: what does it mean for the root of the tree to be `undefined`?

Reply:
This should've been removed. As @CraigMacomber mentioned in the previous comment. I will apply it right now.